damnit/backend/supervisord.py

```diff
@@ -162,6 +162,7 @@ def initialize_and_start_backend(root_path, proposal=None):
         # Initialize database
         db = DamnitDB.from_dir(root_path)
         db.metameta["proposal"] = proposal
+        db.metameta["context_python"] = "/gpfs/exfel/sw/software/mambaforge/22.11/envs/202401/bin/python"
```

Comment:
I think we almost always now run the context file in our `exfel-python` environment rather than the environment where DAMNIT is installed, so that we can use tools like EXtra. So this makes that the default.

We could add more UI to let people choose the environment, but I don't want to spend time on that if there isn't a clear need.